revenue_app/utils.py

```diff
@@ -565,24 +556,44 @@ def sales_flag_summary(trx_currencies, filter, usd=None):
     return json
 
 
-def get_charts_data(transactions, type, filter, usd=None):
+def get_charts_data(transactions, type, filter):
+    ref_currency = 'local_currency' if 'local_currency' in transactions.columns else 'currency'
     trx_currencies = {
-        'Argentina': transactions[transactions['currency'] == 'ARS'],
-        'Brazil': transactions[transactions['currency'] == 'BRL'],
+        'Argentina': transactions[transactions[ref_currency] == 'ARS'],
+        'Brazil': transactions[transactions[ref_currency] == 'BRL'],
     }
     json = {}
     if type == 'payment_processor':
-        json = payment_processor_summary(trx_currencies, filter, usd)
+        json = payment_processor_summary(trx_currencies, filter)
     elif type == 'sales_flag':
-        json = sales_flag_summary(trx_currencies, filter, usd)
+        json = sales_flag_summary(trx_currencies, filter)
     return json
 
 
-def convert_dataframe_to_usd(dataframe, usd):
-    if (not usd) or (None in usd.values()):
-        return dataframe
-    for currency in dataframe['currency'].unique():
-        dataframe.loc[dataframe['currency'] == currency, MONEY_COLUMNS] = \
-            dataframe[dataframe['currency'] == currency][MONEY_COLUMNS] / float(usd[currency])
-    dataframe['currency'] = 'USD'
-    return dataframe
+def dataframe_to_usd(transactions, exchange_data):
+    trx = []
```

Comment:
nit: more meaningful variable name. something like `transactions_by_month` may be